src/views/contributors.ts

```diff
@@ -66,14 +72,26 @@ export class ContributorsView extends BlockView<Trackable<ContributorsNode>> {
     }
     this.version = affs.version
     this.container.innerHTML = ''
+
+    const comment = createCommentMarker('div', this.node.attrs.id)
```

Comment:
Aren't we creating a marker here and not the comment? And since we are controlling the creation of it, do we then need to query it later in this.selectNode and this.updateSelection? Maybe we can store it in this.commentMarker, for example, to access it without querying, since we already have a reference on it?